### ISSUE TITLE
Stars don't work as wildcards in tooling

### ISSUE THREAD

**Issue by neclimdul** [lando/lando#2280]
## Summary 

Testing with the latest edge version of lando (v3.0.3) it seems that * isn't treated like &, | and others and the shell doesn't treat it as a wild card.

The following tooling did not work as expected. 

## Demonstration

.lando.yml
```yaml
name: tooling_test
recipe: drupal8
tooling:
  failing:
      cmd: cp -av /app/* /tmp/
  working:
      cmd: cp -av /app/* /tmp/ && echo 'here'
```

```shell
$ touch text.txt
$ lando failing 
cp: cannot stat '/app/*': No such file or directory
$ lando working
'/app/text.txt' -> '/tmp/text.txt'
here
```

# Expected
The example is a bit over simplified for real world but the two commands should behave the same and copy all the files in the directory.

**Comment by neclimdul** [comment]
I ended up working around this by putting the commands in a script. I ended up having to do that anyways to share the logic between two toolings so no big deal but figured I drop the bug so someone else doesn't run into trouble :-D

**Comment by smutlord** [comment]
@neclimdul i think we should probably remove some legacy handling we had for tooling and now just wrap all tooling commands in `/bin/sh -c` by default

**Comment by pirog** [comment]
alright @neclimdul this should be fixed in `3.0.7` when its released
